doc/src/manual/multi-threading.md

```diff
@@ -73,19 +73,20 @@ can be observed from multiple threads. For example, in most cases you should
 use the following code pattern:
 
 ```julia-repl
-julia> lock(a) do
+julia> lock(l) do
```

Comment:
Maybe use `lk` instead of `l`? It looked like 1 to me and I was quite puzzled.

Reply:
good point! I've updated that